src/main/java/com/google/error_analyzer/backend/StoreLogs.java

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.error_analyzer.backend;
+
+import com.google.error_analyzer.backend.LogDao;
+import java.io.IOException;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+
+//This class contains the methods used for processing logs and storing them to the database.
+//This includes removal of special characters, conversion into json string.
+public class StoreLogs {
+
+    private static final Logger logger = LogManager.getLogger(StoreLogs.class);
+    public DaoInterface logDao = new LogDao();
+
+    //convert the logText and logLineNumber to a json string
+    public String convertToJsonString(String logText, int logLineNumber) {
```

Comment:
Optional
Can we make it private?
Can we test it as a part of checkAndStoreLogs?